src/libcadet/CMakeLists.txt

```diff
@@ -218,7 +222,7 @@ if (LAPACK_FOUND)
 	set(LIB_LAPACK_DEFINE "CADET_LAPACK_TRAILING_UNDERSCORE")
 
 	# Add the build target for the static nonlinalg library
-	add_library(libcadet_nonlinalg_static STATIC ${LIBCADET_NONLINALG_SOURCES} ${LIBCADET_NONLINALG_SPARSE_SOURCES})
+	add_library(libcadet_nonlinalg_static STATIC ${LIBCADET_NONLINALG_SOURCES} ${LIBCADET_NONLINALG_SPARSE_SOURCES} "model/binding/LangmuirLDFCBinding.cpp")
```

Comment:
Why is this necessary? Please revert this. The only place for adding the files should be `LIBCADET_BINDINGMODEL_SOURCES`.

Reply:
I am not sure, why I did that, but I have removed it now.